attestation-agent/lib/src/token.rs

```diff
@@ -47,3 +55,22 @@ pub(crate) async fn get_kbs_host_from_cmdline() -> Result<String> {
 
     Ok(kbs_host)
 }
+
+pub(crate) async fn get_kbs_host_from_config_file() -> Result<String> {
+    // We only care about the aa_kbc_params value at the moment
+    #[derive(Debug, Deserialize)]
+    struct AgentConfig {
+        aa_kbc_params: Option<String>,
+    }
+
+    // Hard-code agent config path to "/etc/agent-config.toml" as a workaround
+    let agent_config_str = fs::read_to_string("/etc/agent-config.toml")
+        .expect("Failed to read /etc/agent-config.toml file");
```

Comment:
`expect` here will cause the process panic when error occurs, which is like `unwrap`. We do not encourage panic inside.

We can import `anyhow::Context` to convert any kinds of error into `anyhow` error with a context information.

```suggestion
        .context("Failed to read /etc/agent-config.toml file")?;
```